src/html.pre.js

```diff
@@ -81,8 +81,9 @@ async function pre(context, action) {
   let desc = [];
   document.querySelectorAll('div > p').forEach((p) => {
     if (desc.length === 0) {
-      const words = p.textContent.trim().split(/\s+/);
-      if (words.length >= 10) {
+      const text = p.textContent.trim();
+      const words = text.split(/\s+/);
+      if (words.length >= 10 || (words.length === 1 && text.length > 25 && !text.startsWith('http'))) {
```

Comment:
Wouldn't it be safer to (also) check for double-byte characters or if the `charCode` is outside of the latin character range?

Reply:
In the end, it turns out to be much more complicated than expected: finding the number of words in JP is not easy. And they also have mix characters set which also won't work without more magic: 

```
Adobe XDの強力な機能である「スタック」と「パディング」について佐々木雄平さんが解説します。
```

Only reliable solution I see is to ask them to use the Metadata block.